### PR TITLE
Ebonilla/sy 735 store legend position with line plot state

### PR DESCRIPTION
**Pull request by emilbon99**
# Feature Pull Request Template

## Key Information

- [Linear Issue]()

## Description

Maximum 2-3 sentence description describing the changes.

## Basic Readiness Checklist

- [ ] I have performed a self-review of my code.
- [ ] I have added relevant tests to cover the changes.
- [ ] I have updated user facing documentation accordingly.

## Manual QA Additions

- [ ] I have updated the [Release Candidate](/.github/PULL_REQUEST_TEMPLATE/rc.md) template
with necessary manual QA steps to test my change.

## Breaking Changes

Please list any breaking changes to public or internal packages.

### CODE REVIEW

File: console/src/persist/state.ts [hunk 1]
```diff
@@ -98,6 +98,7 @@ export const open = async <S extends RequiredState>({
   let state = (await db.get<S>(persistedStateKey(version))) ?? undefined;
   if (state != null && migrator != null) {
     try {
+      console.log("applying migrations");
```
Comment (pjdotson) [inline console/src/persist/state.ts:101]: Should this console.log still be here?

File: console/src/persist/state.ts [hunk 2]
```diff
@@ -98,6 +98,7 @@ export const open = async <S extends RequiredState>({
...
       state = migrator(state);
     } catch (e) {
       console.error("unable to apply migrations. continuing with undefined state.");
```
Comment (pjdotson) [inline console/src/persist/state.ts:104]: Should this error appear in the console log or as a notification in the Synnax console?

Comment (emilbon99) [reply console/src/persist/state.ts:104]: Can't make this a notification considering it sets up the code that enables us to display notifications

File: console/src/persist/state.ts (hunk 1 above)
Comment (emilbon99) [reply console/src/persist/state.ts:101]: I'll remove this one

**Pull request merged**